vendor/github.com/docker/go-connections/tlsconfig/config.go

```diff
@@ -13,6 +13,9 @@ import (
 	"os"
 
 	"github.com/Sirupsen/logrus"
+	"github.com/docker/notary"
```

Comment:
We should probably not vendor notary into `go-connections`, because notary also depends upon `go-connections` :)  I think the goal is probably for `go-connections` to have as few dependencies as possible.